play/src/pusher/controllers/BaseHttpController.ts

```diff
@@ -1,7 +1,8 @@
+import { MonitoringInterface } from "../../services/MonitoringInterface";
 import type { Server } from "hyper-express";
 
 export abstract class BaseHttpController {
-    constructor(protected app: Server) {
+    constructor(protected app: Server, protected monitoringInterface: MonitoringInterface) {
```

Comment:
Same thing everywhere in the code: rename `monitoringInterface` to `monitoring`.

```suggestion
    constructor(protected app: Server, protected monitoring: MonitoringInterface) {
```